nengo/conftest.py

```diff
@@ -210,12 +231,17 @@ def pytest_runtest_setup(item):
                 ('logger', 'logs', "logs not requested")]:
             if fixture_name in item.fixturenames:
                 if item.config.getvalue(option):
-                    skip = False
+                    break
                 else:
                     skipreasons.append(message)
-        if skip:
+        else:
             pytest.skip(" and ".join(skipreasons))
 
+    if 'Simulator' in item.fixturenames:
+        for test, reason in _Simulator.unsupported:
```

Comment:
No, probably not. I'm guessing we'd need at least an order of magnitude more tests before this would even be noticeable (I mean even if we did everything here perfectly optimally), so no point sinking time into it.